src/DependencyInjection/Configuration.php

```diff
@@ -63,8 +63,8 @@ public function getConfigTreeBuilder(): TreeBuilder
                             ->scalarNode('collection_name')->end()
                             // MongoDB
                             ->scalarNode('database_name')->end()
-                            ->booleanNode('bulk_write')->end()
-                            ->scalarNode('vector_field_name')->end()
+                            ->scalarNode('vector_field_name')->defaultValue('vector')->end()
+                            ->booleanNode('bulk_write')->defaultValue(false)->end()
```

Comment:
another drawback of that current config structure - these defaults will be present and potentially confusing when using `debug:config llm_chain`

Reply:
don't have a better idea yet